src/ConnectorRuntime.ts

```diff
@@ -42,6 +42,8 @@ export class ConnectorRuntime extends Runtime<ConnectorRuntimeConfig> {
         return this.runtimeConfig.debug ? "debug" : "production";
     }
 
+    public isDebugMode = (): boolean => this.runtimeConfig.debug;
```

Comment:
why do we have the debug mode? please use this instead.

Reply:
I even put it directly below it and didn't notice :D